### PR TITLE
Add new styles to charts

### PR DESCRIPTION
**Pull request by laurajaime**
#### :tophat: What? Why?
- Apply styles to survey results

TODO:
- [x] Styles to conditional questions
- [x] Styles to title and description questions

image

### CODE REVIEW

File: app/helpers/decidim/survey_results/survey_results_helper.rb [hunk 1]
```diff
@@ -5,15 +5,47 @@ module SurveyResults
...
 
+      def conditioned_question?(question)
+        Decidim::Forms::DisplayCondition.where(decidim_question_id: question).present?
+      end
+
+      def title_and_description_question?(question)
+        question.question_type == "title_and_description"
+      end
+
+      def render_participants_icon
```
Comment (tramuntanal) [inline app/helpers/decidim/survey_results/survey_results_helper.rb]: can we move the svg into a `_participants_icon.html.erb` patial and use the standard `render :participants_icon`?

File: app/views/decidim/survey_results/survey_results/_questions.html.erb [hunk 2]
```diff
@@ -1,13 +1,28 @@
...
       </div>
-      <div class="answers-results">
-        <%= render_question_results(full_questionnaire, question) %>
-      </div>
-    </div>
-  <% end %>
-</div>
+      <% if title_and_description_question?(question) %>
+        <p class="subtitle"><%= ActionView::Base.full_sanitizer.sanitize(translated_attribute(question.description)) %></p>
+      <% end %>
```
Comment (tramuntanal) [inline app/views/decidim/survey_results/survey_results/_questions.html.erb]: why not move this block into `app/views/decidim/survey_results/survey_results/_title_and_description.html.erb` and treat this questions the normal way here?

File: config/locales/es.yml [hunk 3]
```diff
@@ -15,9 +15,11 @@ es:
             percentage: Porcentaje
             quantity: Cantidad
         questions:
-          question: Pregunta
+          conditioned_question: "Pregunta %{number} (PREGUNTA CONDICIONADA)"
+          question: "Pregunta %{number}"
         show:
-          num_participants: Un total de %{count} usuarios han respondido la encuesta
+          alert: Estas viendo los resultados de la encuesta.
```
Comment (tramuntanal) [inline config/locales/es.yml]:
```suggestion
          alert: Estás viendo los resultados de la encuesta.
```

**Review by tramuntanal — COMMENTED** [review]
Good!
Small refactorings only

**Comment by laurajaime** [comment]
@tramuntanal can you review it again?

**Review by tramuntanal — APPROVED** [review]
Thank you for the changes @laurajaime , LGTM!

**Pull request merged**